php-checks/src/main/java/org/sonar/php/checks/formatting/ExtendsImplementsLineCheck.java

```diff
@@ -45,7 +45,7 @@ private void checkExtendsAndImplementsLine(FormattingStandardCheck formattingChe
     String msg = getIssueMessage(isExtendsOnClassNameLine, isImplementsOnClassNameLine);
 
     if (msg != null) {
-      formattingCheck.reportIssue("Move " + msg + " to the same line as the declaration of its class name, \"" + className + "\".", node);
+//      formattingCheck.reportIssue("Move " + msg + " to the same line as the declaration of its class name, \"" + className + "\".", node);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)